frontend/src/App.tsx

```diff
@@ -18,15 +19,33 @@ const App = () => {
 			if (response.success) {
 				setUsers(response.data as userData[]);
 			} else {
+				throw Error("Cannot fetch"); // temporary
 				// create error notification wth Chakra
 			}
 		};
 
 		getUsers();
-		console.log("users", users);
-	});
+		console.log("users", users); // this should be deleted later to avoid non stop render
+	}, [users]);
 	// ***PURELY FOR EXAMPLE***
 
+	useEffect(() => {
+		// open browser console to see result
+		const getUser = async () => {
+			const response = await getSingleUser();
+
+			if (response.success) {
+				setSingleUser(response.data as userData[]);
+			} else {
+				throw Error("Cannot fetch"); // temporary
+				// create error notification wth Chakra
+			}
+		};
+
+		getUser();
+		console.log("users", singleUser); // this should be deleted later to avoid non stop render
+	}, [singleUser]);
+
```

Comment:
You've done well with this @ifasama but the front end implementation was not part of this ticket (Noah is currently working on it). This ticket was solely concerned with creating the endpoint. Please remove any front end changes